src/test/resources/com/puppycrawl/tools/checkstyle/utils/javadocutil/InputJavadocUtilTest_ContentOfInlineCustomTag.java

```diff
@@ -0,0 +1,21 @@
+package com.puppycrawl.tools.checkstyle.utils.javadocutil;
+/* Config: Default
+ */
+
+/**
+ *
+ * {@summary This is the summary of the class.
+ * <ul>
+ * <li>This class does great things.</li>
+ * <li>This class is a test class.</li>
+ * </ul>}
+ */
+public class InputJavadocUtilTest_ContentOfInlineCustomTag { // ok
```

Comment:
usually, naming is like `InputJavadocUtilInlineCustomTag` or something like this. No underscore, and short "description"

Please also add a bit more inputs or extend this one.

Reply:
> usually, naming is like Inp......

This PR's quite old, don't remember the rationale for giving it such a complicated name, changed now and added some more inputs